### PR TITLE
Redesign send search

### PR DESCRIPTION
**Pull request by musidlo**

### CODE REVIEW

File: packages/app/components/SearchBar.tsx [hunk 1]
```diff
@@ -477,16 +506,32 @@ function Search({ label, placeholder = 'Sendtag, Phone, Send ID, Address' }: Sea
...
+                focusStyle: {
+                  boc: borderColor,
+                  bw: 1,
+                  outlineWidth: 0,
+                },
+                iconBefore: <IconSearch color={'$silverChalice'} />,
+                iconAfter: query && (
+                  <Stack onPress={handleClearClick} cursor={'pointer'}>
+                    <IconX color={'$silverChalice'} size="$1" />
+                  </Stack>
```
[the remark this answers is not in the document]
Comment (musidlo) [reply packages/app/components/SearchBar.tsx]: alright, you are the boss here, changed it to be a button

**Comment by youngkidwarrior** [comment]
image

Light mode is a little hard to read. Let's darken the placeholder text

**Comment by musidlo** [comment]
Hey @youngkidwarrior, darkened placeholder color on light mode, also as Ethen asked, I removed italics from placeholder, bumped font size of it and modified search icon

File: packages/app/features/send/components/SendTopNav.tsx [hunk 2]
```diff
@@ -57,7 +57,7 @@ export function SendTopNav() {
                 case Boolean(sendParams.recipient):
                   return 'Enter Amount'
                 default:
-                  return 'Select Recipient'
+                  return 'Send'
```
Comment (youngkidwarrior) [inline packages/app/features/send/components/SendTopNav.tsx:60]: One more thing.

 hide the Send header on desktop. Align the search bar with top of other screens

**Review by youngkidwarrior — CHANGES_REQUESTED** [review]

**Review by youngkidwarrior — APPROVED** [review]

**Pull request merged**